### ISSUE TITLE
Could not find remote builds

### ISSUE THREAD

**Issue by Rosencratz** [remyroy/CDDA-Game-Launcher#269]
Hi, new to CDDA and using launchers, but I've been checking on the issues logs to find that it's nto the first time this is happening. As far as I can tell, it's beyond something the dev of the launcher could maintain, but thought I'd still post this to let you be aware the current up to date version still suffers this issue. At first I thought it was because I am in China using a Chinese locale laptop, but it seems like even with a VPN on, it doesn't make a difference. Which got me checking here to figure out what to do. 

![image](https://user-images.githubusercontent.com/49988651/56744587-5da9fa00-67ab-11e9-9e80-3a2fd113a20f.png)

**Comment by remyroy** [comment]
Can you access https://dev.narc.ro/cataclysm/jenkins-latest/Windows_x64/Tiles/ ? This is where the launcher gets the available builds.

**Comment by Rosencratz** [comment]
> Can you access https://dev.narc.ro/cataclysm/jenkins-latest/Windows_x64/Tiles/ ? This is where the launcher gets the available builds.

Yes. I've been downloading the new versions manually since I've heard of the launcher through that site. That said, the download is incredibly slow. Roughly 30-40kb/s

**Comment by remyroy** [comment]
The launcher performs an HTTPS request to https://dev.narc.ro/cataclysm/jenkins-latest/Windows_x64/Tiles/ to get the available builds. If you can access that URL, the launcher should also be able to.

**Comment by Rosencratz** [comment]
> The launcher performs an HTTPS request to https://dev.narc.ro/cataclysm/jenkins-latest/Windows_x64/Tiles/ to get the available builds. If you can access that URL, the launcher should also be able to.

Strange. I can't say I know what's going on in that case. The website opens fine for me, even if a wee bit slow. Does the launcher have problems if the internet connection is very slow?

**Comment by ClOxIon** [comment]
When I run it via python console, it says

**Comment by KurzedMetal** [comment]
Does this still happen in the latest versoins?
Commit https://github.com/remyroy/CDDA-Game-Launcher/commit/2797a7b418d55c65b16cafc0d467f0c4d0b33a45 changed how CDDA Launcher gets the builds (now gets them from github), and could've potentially workarounded this issue with dev.narc.ro.

**Comment by squiddy555** [comment]
I updated launcher to 1.4.1 today, and it still fails to find remote builds.
![screenshot_19-06-15_21-40-29](https://user-images.githubusercontent.com/51860272/59554004-96e63580-8fb6-11e9-9ff2-7b12242d80de.png)
I can access https://dev.narc.ro/cataclysm/jenkins-latest/Windows_x64/Tiles/ just ok, and I can download builds manually as well. If that matters, I tried using vpn, but no success.

**Comment by KurzedMetal** [comment]
@squiddy555 That seems like a networking issue when grabbing the builds (probably between your ISP and github servers). I've been checking the build list in the last several mins (just like you) and I do see the builds.

Also, the launcher don't get the builds from dev.narc.ro anymore, it gets them from https://api.github.com/repos/CleverRaven/Cataclysm-DDA/releases

Try to access that page manually, it should be a JSON with a list of releases.

**Comment by squiddy555** [comment]
That's what I see at that page.
![screenshot_19-06-15_22-16-58](https://user-images.githubusercontent.com/51860272/59554377-53da9100-8fbb-11e9-938d-137261b717b2.png)

**Comment by KurzedMetal** [comment]
And you still have the issue? That's weird

**Comment by nizhuan-jjr** [comment]
I also have this issue. I can access https://api.github.com/repos/CleverRaven/Cataclysm-DDA/releases and https://dev.narc.ro/cataclysm/jenkins-latest/Windows_x64/Tiles/, yet the launcher says 

> Could not find remote builds [HTTP None](None) when requesting https://api.github.com/repos/CleverRaven/Cataclysm-DDA/releases

**Comment by squiddy555** [comment]
@nizhuan-jjr thanks, worked like a charm! In my case it was cyrillic symbols in path name, moved folder to C:/ and everything is fine again.
Very wierd issue though.
